test/system/src/main/java/io/pravega/test/system/framework/TestExecutorFactory.java

```diff
@@ -22,21 +25,28 @@
     public enum TestExecutorType {
         LOCAL,
         DOCKER,
+        K8s,
         REMOTE_SEQUENTIAL,
         REMOTE_DISTRIBUTED //TODO: https://github.com/pravega/pravega/issues/2074.
     }
 
-    public TestExecutor getTestExecutor(TestExecutorType type) {
+    TestExecutor getTestExecutor(TestExecutorType type) {
         switch (type) {
             case DOCKER:
                 return getDockerExecutor();
             case REMOTE_SEQUENTIAL:
                 return getMarathonSequentialExecutor();
+            case K8s:
+                 return new K8SequentialExecutor();
```

Comment:
Is there any reason why this executor is not obtained through a Getter like the Docker and Marathon ones?

E.g.:
```
    @Getter(lazy = true)
    private final TestExecutor kubernetesSequentialExecutor = new KubernetesBasedTestExecutor();
...
            case KUBERNETES:
                 return getKubernetesSequentialExecutor();
```

Reply:
updated the code to follow the same pattern.